src/functions.c

```diff
@@ -821,49 +827,50 @@ void functionFlushCommand(client *c) {
     /* Indicate that the command changed the data so it will be replicated and
      * counted as a data change (for persistence configuration) */
     server.dirty++;
-    addReply(c,shared.ok);
+    addReply(c, shared.ok);
 }
 
 /* FUNCTION HELP */
 void functionHelpCommand(client *c) {
     const char *help[] = {
-"LOAD [REPLACE] <FUNCTION CODE>",
-"    Create a new library with the given library name and code.",
-"DELETE <LIBRARY NAME>",
-"    Delete the given library.",
-"LIST [LIBRARYNAME PATTERN] [WITHCODE]",
-"    Return general information on all the libraries:",
-"    * Library name",
-"    * The engine used to run the Library",
-"    * Functions list",
-"    * Library code (if WITHCODE is given)",
-"    It also possible to get only function that matches a pattern using LIBRARYNAME argument.",
-"STATS",
-"    Return information about the current function running:",
-"    * Function name",
-"    * Command used to run the function",
-"    * Duration in MS that the function is running",
-"    If no function is running, return nil",
-"    In addition, returns a list of available engines.",
-"KILL",
-"    Kill the current running function.",
-"FLUSH [ASYNC|SYNC]",
-"    Delete all the libraries.",
-"    When called without the optional mode argument, the behavior is determined by the",
-"    lazyfree-lazy-user-flush configuration directive. Valid modes are:",
-"    * ASYNC: Asynchronously flush the libraries.",
-"    * SYNC: Synchronously flush the libraries.",
-"DUMP",
-"    Return a serialized payload representing the current libraries, can be restored using FUNCTION RESTORE command",
```

Comment:
> I would prefer that the formatting be done by the developer prior to commit, and any time the formatter actually changes something it fails that stage of the pipeline.

Yeah that's how it will work once we've formatted all the old code.

We should fix long lines like that manually, preferably in separate PRs before we autoformat everything.